web/Weblate-server.qmd

```diff
@@ -205,26 +206,67 @@ rebase refs/remotes/origin/trunk: command returned error: 1
 Weblate will provide a suggestion on how to fix, but in short:
 
 1. SSH to the Weblate server
-2. Attach the Docker container:
+2. Make a backup on the Git repo before messing with the rebase process:
+
+    ```sh
+    tar -zcvf /root/base-r-gui-repo-$(date '+%Y-%m-%d').tar.gz \
+      /var/lib/docker/volumes/weblate-docker_weblate-data/_data/vcs/r-project/base-r-gui
+    ```
+
+3. Attach the Docker container:
 
     ```sh
     sudo docker exec -ti weblate-docker_weblate_1 bash
```

Comment:
where does the Dockerfile for this live BTW?

Reply:
it's the standard `weblate` img as per https://contributor.r-project.org/translations/Weblate-server.html#docker-images-and-configuration

Reply:
we could create our own based on this and preinstall things .. once we know what is required :)

Reply:
Got it, hopefully the above will tip that off to future readers.